Contest_API.md

```diff
@@ -610,13 +610,32 @@ The following endpoint is associated with API information:
 
 Properties of version object:
 
+| Name         | Type              | Description
+| :----------- | :---------------- | :----------
+| version      | string            | Version of the API. For this version must be the string `draft`. Will be of the form `<yyyy>-<mm>`, `<yyyy>-<mm>-draft`, or simply `draft`.
+| version\_url | string            | Link to documentation for this version of the API.
+| provider     | provider object ? | Information about the data provider
+
+Properties of the provider object:
```

Comment:
It's valid to add a provider that's completely empty, which seems wrong. Maybe at least name should be required?